components/bt/host/bluedroid/btc/profile/std/spp/btc_spp.c

```diff
@@ -895,7 +895,12 @@ static void btc_spp_write(btc_spp_args_t *arg)
             }
         } else {
             if (fixed_queue_enqueue(slot->tx.queue, arg->write.p_data, 0)) {
-                BTA_JvRfcommWrite(arg->write.handle, slot->id, arg->write.len, arg->write.p_data);
+                if (BTA_JvRfcommWrite(arg->write.handle, slot->id, arg->write.len, arg->write.p_data) == BTA_JV_SUCCESS) {
```

Comment:
I think other places that call `BTA_JvRfcommWrite` should also be dealt with accordingly.